templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+                                        <a class="text-primary" href="./forms_admin.php?id={{ row.id|escape('url') }}&method={{ row.statusMethod|escape('url') }}">{{ row.status|translate }}</a>
+                                    {% endif %}
+                                </td>
+                                <td>{{ row.packageStatus }}</td>
+                                <td class="{{ row.dbStatusClass }}">
+                                    {% if row.dbStatus == "DB Installed" %}
+                                        {{ row.dbStatus|translate }}
+                                    {% else %}
+                                        <i class="fa fa-fw fa-database"></i>&nbsp;<a class="text-danger" href="./forms_admin.php?id={{ row.id|escape('url') }}&method=install_db">{{ row.dbStatus|translate }}</a>
+                                    {% endif %}
+                                </td>
+                                <td><input type='text' size="4" class="form-control input-sm" name='priority_{{ row.id }}' value='{{ row.priority|escape('html_attr') }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='category_{{ row.id }}' value='{{ row.category|escape('html_attr') }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='nickname_{{ row.id }}' value='{{ row.nickname|escape('html_attr') }}'></td>
+                                <td>
+                                    <select name='aco_spec_{{ row.id }}'>
+                                        <option value=''></option>
+                                        {{ aco_options(row.aco)|raw }}
+                                    </select>
+                                </td>
+                            </tr>
+                        {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+                <div role="tabpanel" class="tab-pane" id="unregistered">
+
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>{{ "Name"|translate }}</th>
+                            <th>&nbsp;</th>
+                            <th>{{ "File compressed"|translate }}?</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if unregistered|length == 0 %}
+                            <td colspan="3" class="text-center">{{ "No unregistered forms available"|translate }}</td>
+                        {% else %}
+                            {% for row in unregistered %}
+                            <tr>
+                                <td>{{ row.name }}</td>
+                                <td>
+                                    {% if row.extracted %}
+                                        <a href="./forms_admin.php?name=name&method=register">
```

Comment:
anytime a link opens a script, need to incorporate:
http://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows